### PR TITLE
(Doc+) Sparse Vectors NA to mapping analyzers

### PR DESCRIPTION
**Pull request by stefnestor**
👋 howdy, team! Mini expansion to say Sparse Vectors are NA to mapping analyzers per Dev @demjened @Mikep86 feedback. TIA!

### CODE REVIEW

**Comment by stefnestor** [comment]
That was gross, but fixed target branch, sorry for the noise.

**Review by leemthompo — APPROVED** [review]
LGTM

**Comment by leemthompo** [comment]
run docs-build

File: docs/reference/mapping/types/sparse-vector.asciidoc [hunk 1]
```diff
@@ -91,7 +91,7 @@ NOTE: `sparse_vector` fields can not be included in indices that were *created*
 NOTE: `sparse_vector` fields only support strictly positive values.
 Negative values will be rejected.
 
-NOTE: `sparse_vector` fields do not support querying, sorting or aggregating.
+NOTE: `sparse_vector` fields do not support <<analysis-intro,analyzers>>, querying, sorting or aggregating.
```
Comment (stefnestor) [inline docs/reference/mapping/types/sparse-vector.asciidoc:94]:
```suggestion
NOTE: `sparse_vector` fields do not support <<analysis,analyzers>>, querying, sorting or aggregating.
```

Sorry, mixed up the ES "guide" with "reference" pages. Now will link to [here](https://www.elastic.co/guide/en/elasticsearch/reference/current/analysis.html).

Comment (leemthompo) [reply docs/reference/mapping/types/sparse-vector.asciidoc]: It can all get very 😵‍💫 fast :)

**Pull request merged**